planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -734,14 +725,21 @@ void NormalLaneChange::calcTurnSignalInfo()
     const auto diff_length = std::abs(current_twist.linear.x) * diff_time;
     return get_blinker_pose(path.path, diff_length);
   });
-  turn_signal_info.desired_end_point = path.shift_line.end;
 
+  // desired end pose
+  const auto & length_ratio =
```

Comment:
```suggestion
  const auto length_ratio =
```

this returns trivial type, so you dont need `&`